### ISSUE TITLE
Add modifier to choose serialization format

### ISSUE THREAD

**Issue by albertmeronyo** [CLARIAH/COW#68]

**Comment by albertmeronyo** [comment]
Fixed in a1027fd866944aaa2718e011f296b2d058830cb6

**Comment by rlzijdeman** [comment]
hi @albertmeronyo , in case of .ttl the jinja magic appears to be ignored when outputting as .ttl. Look for the _row_ and _adamBUri1853_ that aren't 'expanded'. Testing the same .csv and meta-data.json on Cattle /does/ provide the expected output.

```
<https://iisg.amsterdam/anyUri.csv> dc:license <http://opendefinition.org/licenses/cc-by/> ;
 72     dc:modified "2019-02-12"^^xsd:date ;
 73     dc:publisher [ schema:name "CLARIAH Structured Data Hub - Datalegend"@en ;
 74             schema:url <http://datalegend.net/> ] ;
 75     dc:title "anyUri.csv"@en ;
 76     csvw:dialect [ csvw:delimiter "," ;
 77             csvw:encoding "ascii" ;
 78             csvw:quoteChar "\"" ] ;
 79     csvw:tableSchema [ csvw:aboutUrl <https://iisg.amsterdam/__row_> ;
 80             csvw:column ( <https://iisg.amsterdam/anyUri.csv/column/id> <https://iisg.amsterdam/anyUri.csv/column/buurt1853> [ csvw:aboutUrl <https://iisg.amsterdam/_buurt1853_> ;
 81                         csvw:datatype xsd:anyURI ;
 82                         csvw:name "adamBUri1853" ;
 83                         csvw:propertyUrl owl:sameAs ;
 84                         csvw:valueUrl <https://iisg.amsterdam/_adamBUri1853_> ;
 85                         csvw:virtual true ] <https://iisg.amsterdam/anyUri.csv/column/adamBUri1853> ) ;
 86             csvw:primaryKey "id" ] ;
 87     csvw:url "anyUri.csv"^^xsd:anyURI .
```

**Comment by albertmeronyo** [comment]
Thanks @rlzijdeman Patched in `0.41`
The conversion to the requested format only happens now after everything is serialized first in nquads (the nq file is always preserved, and additionally the requested format is also produced).

**Comment by rlzijdeman** [comment]
Check! New feature described in https://github.com/CLARIAH/COW/wiki/12.-example:-changing-output-format-of-triples,-e.g.-as-turtle-(.ttl)
